lbr_demos/lbr_demos_fri_ros2_cpp/package.xml

```diff
@@ -12,6 +12,12 @@
   <depend>fri_vendor</depend>
   <depend>lbr_fri_msgs</depend>
   <depend>rclcpp</depend>
+  <depend>tf2_ros</depend>
+  <depend>urdf</depend>
+  <depend>geometry_msgs</depend>
+  <depend>sensor_msgs</depend>
+  <depend>orocos_kdl</depend>
```

Comment:
this dependency should be replaced by `orocos_kdl_vendor`

Reply:
okay, I get it, thank you.